### DISCUSSION TITLE
app真的能发帖吗？

### DISCUSSION THREAD

**Discussion by JT9364** [share121/inter-knot#5013] [灌水]
毕竟绳网app找不到ui

**Comment by JT9364** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/1a2616d760a6f334a4a9abca46267af3_7281603850910753959.png" alt="三月七 我" width="75" >

**Comment by JT9364** [comment]
发现了，解决办法就只是在对应浏览器修改为《桌面版网站》就好了（笑

**Comment by passbyDFX** [comment]
哈哈: ）乐
